### PR TITLE
Small improvements

### PR DESCRIPTION
**Pull request by DomGarguilo**

### CODE REVIEW

File: server/manager/src/main/java/org/apache/accumulo/manager/compaction/coordinator/CompactionCoordinator.java [hunk 1]
```diff
@@ -381,9 +374,7 @@ public void run() {
         update.setState(TCompactionState.IN_PROGRESS);
         update.setMessage(RESTART_UPDATE_MSG);
         rc.addUpdate(System.currentTimeMillis(), update);
-        if (!rc.isStartTimeSet()) {
-          rc.setStartTime(this.coordinatorStartTime);
-        }
+        rc.setStartTime(this.coordinatorStartTime);
```
Comment (DomGarguilo) [inline server/manager/src/main/java/org/apache/accumulo/manager/compaction/coordinator/CompactionCoordinator.java:377]: This check already happens within setStartTime

File: server/manager/src/main/java/org/apache/accumulo/manager/compaction/coordinator/CompactionCoordinator.java [hunk 2]
```diff
@@ -1069,8 +1060,10 @@ public void updateCompactionStatus(TInfo tinfo, TCredentials credentials,
         case CANCELLED:
         case FAILED:
         case SUCCEEDED:
-          LONG_RUNNING_COMPACTIONS_BY_RG
-              .getOrDefault(rc.getGroupName(), new TimeOrderedRunningCompactionSet()).remove(rc);
+          var compactionSet = LONG_RUNNING_COMPACTIONS_BY_RG.get(rc.getGroupName());
+          if (compactionSet != null) {
+            compactionSet.remove(rc);
+          }
```
Comment (DomGarguilo) [inline server/manager/src/main/java/org/apache/accumulo/manager/compaction/coordinator/CompactionCoordinator.java:1066]: This patterns avoids creating a new, empty `TimeOrderedRunningCompactionSet()` then immediately calling `.remove()` on it

File: server/base/src/main/java/org/apache/accumulo/server/metrics/MetricResponseWrapper.java [hunk 3]
```diff
@@ -81,11 +82,11 @@ public MetricResponseWrapper(FlatBufferBuilder builder) {
    * Remove tags from the Metric that duplicate other information found in the MetricResponse
    */
   private List<Tag> reduceTags(List<Tag> tags, List<Tag> extraTags) {
-    return Stream.concat(tags.stream(), extraTags.stream()).filter(t -> {
-      return !t.getKey().equals(INSTANCE_NAME_TAG_KEY) && !t.getKey().equals(PROCESS_NAME_TAG_KEY)
-          && !t.getKey().equals(RESOURCE_GROUP_TAG_KEY) && !t.getKey().equals(HOST_TAG_KEY)
-          && !t.getKey().equals(PORT_TAG_KEY);
-    }).collect(Collectors.toList());
+    final Set<String> existingTags = Set.of(INSTANCE_NAME_TAG_KEY, PROCESS_NAME_TAG_KEY,
```
Comment (dlmarion) [inline server/base/src/main/java/org/apache/accumulo/server/metrics/MetricResponseWrapper.java]: We could probably move this out of this method and make it a static final variable, no need to create the Set every time the method is called.

**Review by dlmarion — APPROVED** [review]
Looks good, I think we can make one little change then merge this in

File: server/base/src/main/java/org/apache/accumulo/server/metrics/MetricResponseWrapper.java (hunk 3 above)
Comment (DomGarguilo) [reply server/base/src/main/java/org/apache/accumulo/server/metrics/MetricResponseWrapper.java]: Good idea. Fixed in 9c656f6

**Pull request merged**